apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/config/DataStoreConfig.java

```diff
@@ -33,6 +33,13 @@ public interface DataStoreConfig {
   @WithDefault("${stargate.multi-tenancy.enabled}")
   boolean ignoreBridge();
 
+  /**
+   * @return If call(s) to fetch metadata fail, should we just return default settings ({@code
+   *     true}) or throw an exception ({@code false}). Defaults to {@code false}.
+   */
+  @WithDefault("false")
+  boolean useFallbacksIfCallFails();
```

Comment:
maybe something like `bridgeFallbackEnabled()`, the other one is really hard to get with `-D` or evn vars

Reply:
Yeah, good point. Will change as suggested.